Automattic/jetpack
# Update product cards description on My Jetpack

## Proposed changes:

* Update state copy for some products that are displayed on My Jetpack

### Other information:


## Jetpack product discussion


See linked issue

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






The changes made here are quite simple. If you wanna test the functionality:
* Spin up a JN site with Jetpack Beta pointing to this branch
* Go to My Jetpack and confirm that you can see the new copy
